### PR TITLE
HW2

### PR DESCRIPTION
**Pull request by Jukis252**
First push to main with improved github visualization

### CODE REVIEW

File: CoherentHW2/CoherentHW2_Task1/DiagonalMatrix.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        }
+
+        public int Track()
+        {
+            return DiagonalElementsPublic.Sum();
+        }
+
+        public override bool Equals(object? obj)
+        {
+            var other = obj as DiagonalMatrix;
```
Comment (radislavkireev) [inline CoherentHW2/CoherentHW2_Task1/DiagonalMatrix.cs:48]: Handle the case when obj is not a type of DiagonalMatrix

File: CoherentHW2/CoherentHW2_Task2/Entities/Activitys.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace HW2_Task2.Entities
+{
+    internal abstract class Activitys
```
Comment (radislavkireev) [inline CoherentHW2/CoherentHW2_Task2/Entities/Activitys.cs]: typo Activiti**e**s

File: CoherentHW2/CoherentHW2_Task2/Entities/PracticalLesson.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿namespace HW2_Task2.Entities
+{
+    internal class PracticalLesson : Activitys
+    {
+        private string Description { get; set; }
+        private string TaskCondition { get; set; }
+        public string TaskSoliution { get; set; }
```
Comment (radislavkireev) [inline CoherentHW2/CoherentHW2_Task2/Entities/PracticalLesson.cs]: typo TaskSol**ut**ion

**Review by radislavkireev — APPROVED** [review]

**Review by Jukis252 — COMMENTED** [review]
Fixed typos

**Pull request merged**